ONBOARDING.md

```diff
@@ -0,0 +1,121 @@
+- Suivre la [doc du back](https://github.com/tournesol-app/tournesol/tree/main/backend)
+- Installer les dépendances de test dans les dossier `tests`
+- Faire le Setup d’une clé API pour discuter avec l’API Youtube
+
+### Front
+
+- Installer nvm 
+- Installer la version 14 `nvm install 14` et puis ` npm install -global yarn`
+- Créer un envrionement virtuel `mkvirtualenv {nom}`
+- Suivre la section Installation du front
+
+## Tips
+
+- Pour le backend, quand une modification est faite, il faut (toujours) écrire un test ou préciser un test existant. La commande `pytest` permet de lancer tous les tests.
+Cependant la commande `pytest --html=report.html –self-contained-html` permet de créer un fichier de rapport propre avec le détail du log séparé pour chacun des tests.
+
+- Pour mettre à jour le schéma d’OpenApi et plus précisément le `.yaml`, il faut d’abord lancer le back, puis dans le dossier du front tapez la commande `yarn run update-schema`.
+
+- Pour mettre à jour les fichiers dépendant du .yaml d’OpenApi, il faut lancer la commande suivante le dossier `scripts` du front : ` ./generate-services-from-openapi.sh`.
+
+- Pour corriger le style du code de manière automatique, il est possible d’activer un plugin selon l’IDE. Sinon on la commande `yarn lint:fix` permet dans tous les cas de corriger le code de manière automatique.
+
+# Fonctionnement de MR/projet
+
+Le suivi du projet et des tâches en cours se fait sur ce [board](https://github.com/tournesol-app/tournesol/projects/9)
+
+Il est possible de créer des idées qui nous semblent pertinentes en éditant une issue et en la plaçant dans la colonne `Backlog and ideas`.
+
+La core-team décide par la suite des tâches à prioriser et dépose les tickets correspondants dans la colonne `Ready`. On peut s’assigner sur un ticket après avoir discuté avec [Adrien][Adrien], [Romain][Romain]. ou [Louis][Louis].
+
+Quand votre travail est assez avancé vous pouvez créer une PR de votre branche vers `main` sous Github. Vous pouvez aussi demander à [Adrien][Adrien] ou [Romain][Romain] de relire votre PR mais si non totalement aboutie (PR en statut draft) pour avoir quelques feedbacks et potentiellement modifier certaines choses.
+
+Quand votre PR vous semble dans un bon état pour être mergé, demandez la revue d’[Adrien][Adrien], [Romain][Romain] et [Louis][Louis] (au moins). L’opération de merge ne peut être effectuée que si au moins une personne approuve votre PR.
+
+Un changement dans le back doit impliquer des tests pour contrôler le comportement attendu, ainsi que les possibles comportements qui doivent être évités.
+
+Côté front, une modification peut aussi s’accompagner de tests pour le component dédié.
+
+La création de test end-to-end n’est pas systématique et il est bon de voir avec [Adrien][Adrien] et [Romain][Romain] pour savoir l’utilité d’un nouveau test.
+
+Pour se tenir au courant des avancées du projet, des bugs, rejoignez le [discord][tournesol-discord-join]
```

Comment:
This part is going to be translated in another document called `Contributing.md`